src/base/run_state.ml

```diff
@@ -83,13 +83,14 @@ let get_variable_value { num_inputs; input; aux; _ } : int -> 'field =
   if i <= num_inputs then Vector.get input (i - 1)
   else Vector.get aux (i - num_inputs - 1)
 
-let store_field_elt { next_auxiliary; aux; _ } x =
-  let v = !next_auxiliary in
-  incr next_auxiliary ; Vector.emplace_back aux x ; Cvar.Unsafe.of_index v
+let store_field_elt { next_auxiliary; aux; _ } x = failwith "todo"
+  [@@ocaml.warning "-27"]
+(* let v = !next_auxiliary in
+   incr next_auxiliary ; Vector.emplace_back aux x ; Cvar.Unsafe.of_index v *)
 
-let alloc_var { next_auxiliary; _ } () =
-  let v = !next_auxiliary in
-  incr next_auxiliary ; Cvar.Unsafe.of_index v
+let alloc_var { next_auxiliary; _ } () = failwith "todo" [@@ocaml.warning "-27"]
+(* let v = !next_auxiliary in
+   incr next_auxiliary ; Cvar.Unsafe.of_index v *)
```

Comment:
need to figure this one out, next PR will abstract run_state and remove that hopefully

Reply:
Likewise, `alloc_var` on the constraint system.

Reply:
this is now implemented in rust